test/utils.go

```diff
@@ -52,12 +52,18 @@ func ExpectResourceCreated(c client.Client, obj client.Object) {
 }
 
 // ExpectResourceDestroyed asserts that the given object no longer exists.
-func ExpectResourceDestroyed(c client.Client, obj client.Object) {
+func ExpectResourceDestroyed(c client.Client, obj client.Object, timeout time.Duration) {
 	var err error
 	EventuallyWithOffset(1, func() error {
 		err = GetResource(c, obj)
-		return err
-	}, 10*time.Second).ShouldNot(BeNil(), fmt.Sprintf("GetResource %s should return error", obj.GetName()))
+		if errors.IsNotFound(err) || errors.IsGone(err) {
+			return nil
+		} else if err != nil {
+			return err
+		} else {
+			return fmt.Errorf("Resource \"%s\" should no longer exist", obj.GetName())
```

Comment:
You can use single quotes here mixed with the double quotes to avoid needing to escape the string.

Reply:
Fixed.